concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -0,0 +1,342 @@
+Luangthongkum-2019-341-213	213	path, trail, way	2252	PATH
+Luangthongkum-2019-341-214	214	peacock	3382	PEACOCK
+Luangthongkum-2019-341-215	215	pen, coop	3401	HENHOUSE
+Luangthongkum-2019-341-216	216	penis	1222	PENIS
+Luangthongkum-2019-341-217	217	pheasant	3060	PHEASANT
+Luangthongkum-2019-341-218	218	pig	1337	PIG
+Luangthongkum-2019-341-219	219	pile	2984	HEAP
+Luangthongkum-2019-341-220	220	pointed (as of needles), sharp point		
+Luangthongkum-2019-341-221	221	pole	282	POLE
+Luangthongkum-2019-341-222	222	porcupine	2510	PORCUPINE
+Luangthongkum-2019-341-223	223	porcupine (brush-tailed~)		
+Luangthongkum-2019-341-224	224	pot	1462	POT
+Luangthongkum-2019-341-225	225	pound (v.)	3768	POUND (FOOD)
+Luangthongkum-2019-341-226	226	pull up	2066	PULL UP (RICE SEEDLINGS)
+Luangthongkum-2019-341-227	227	pumpkin (ash~)	3444	WAX GOURD
+Luangthongkum-2019-341-228	228	push	1452	PUSH
+Luangthongkum-2019-341-229	229	put on, wear	158	PUT ON
+Luangthongkum-2019-341-230	230	put on (a necklace), wear		
+Luangthongkum-2019-341-231	231	put out (the tongue)		
+Luangthongkum-2019-341-232	232	quail	1893	QUAIL
+Luangthongkum-2019-341-233	233	rabbit, hare	2345	LEPORID (RABBIT OR HARE)
+Luangthongkum-2019-341-234	234	raise, feed (animals)	2134	RAISE (BRING UP)
+Luangthongkum-2019-341-235	235	rattan	2497	RATTAN
+Luangthongkum-2019-341-236	236	reach into		
+Luangthongkum-2019-341-237	237	resin	664	RESIN
+Luangthongkum-2019-341-238	238	right (side)	1019	RIGHT
+Luangthongkum-2019-341-239	239	ripe, cooked	3806	RIPE OR COOKED
+Luangthongkum-2019-341-240	240	roast (by stuffing into fresh bamboo sections)	1962	ROAST (SOMETHING)
+Luangthongkum-2019-341-241	241	root	670	ROOT
+Luangthongkum-2019-341-242	242	rotten, bad smell		
+Luangthongkum-2019-341-243	243	rub, clean, polish, scrub	2235	CLEAN (SOMETHING)
+Luangthongkum-2019-341-244	244	salt	1274	SALT
+Luangthongkum-2019-341-245	245	salty, sweet		
+Luangthongkum-2019-341-246	246	sand	671	SAND
+Luangthongkum-2019-341-247	247	scale (of fish)	266	SCALE
+Luangthongkum-2019-341-248	248	sell	1571	SELL
+Luangthongkum-2019-341-249	249	serow, mountain goat	3270	SEROW
+Luangthongkum-2019-341-250	250	seven	1704	SEVEN
+Luangthongkum-2019-341-251	251	sew	1457	SEW
+Luangthongkum-2019-341-252	252	sharp	1396	SHARP
+Luangthongkum-2019-341-253	253	sheep	1331	SHEEP
+Luangthongkum-2019-341-254	254	shivering	52	SHIVER
+Luangthongkum-2019-341-255	255	shoot (v.)	1172	SHOOT
+Luangthongkum-2019-341-256	256	short, low	2484	SMALL (NOT TALL)
+Luangthongkum-2019-341-257	257	shoulder	1482	SHOULDER
+Luangthongkum-2019-341-258	258	shove (with the foot)		
+Luangthongkum-2019-341-259	259	six	1703	SIX
+Luangthongkum-2019-341-260	260	skink		
+Luangthongkum-2019-341-261	261	sleeping	1585	SLEEP
+Luangthongkum-2019-341-262	262	slippery	3702	SLIPPERY
+Luangthongkum-2019-341-263	263	smell (v.)	1586	SMELL (PERCEIVE)
+Luangthongkum-2019-341-264	264	snail (land~)	1543	SNAIL
+Luangthongkum-2019-341-265	265	snail (water~)		
```

Comment:
Similar as with the lizards above I guess. (and our definition is somehow limited to terrestrial snails even though the gloss "snail" wouldn't make that clear to me).

Reply:
Here the definition of SNAIL clearly points to terrestrial snails so I thought it would be unfair to leave unmapped such a straightforward case.

Reply:
Well our definitions are only there as a convenience and not to be relied upon, the important bit is what is mapped to each entry.